### ISSUE TITLE
full directoryname in the list on the left side?

### ISSUE THREAD

**Issue by DampfHans** [oklai/koala#396]
Hi, I just tested Koala and I really like it. But after adding some projects the left side of the program reads "css, css, css, css...", since the directories with the css-files are always named "css". I think, this is a usual naming. So it would be a great enhancement if the list shows the complete directorynames. I hope you can take this issue into account sometime in the future.
Thanks for this great program!

**Comment by stevesobol** [comment]
I completely agree. Directories in different projects often have the same names. This is essential.
